raiden-dapp/src/components/UpdateSnackbar.vue

```diff
@@ -13,39 +12,32 @@
 <script lang="ts">
 /* istanbul ignore file */
 import { Component, Vue } from 'vue-property-decorator';
+import { mapGetters } from 'vuex';
 import BlurredOverlay from '@/components/overlays/BlurredOverlay.vue';
```

Comment:
Is this still needed after the changes?

Reply:
Fixed by later commit

Reply:
Yes ofc? What do you mean? Why shouldn't it? Else the linter would complain btw...

Reply:
I fail to get a permanlink to the line. But it is just a couple of lines below in the decorator of the class.